.github/workflows/main.yml

```diff
@@ -84,7 +84,7 @@ jobs:
                   npm run test;
             - uses: actions/upload-artifact@v4
               with:
-                  name: pkg ${{ matrix.os }}
+                  name: pkg compat ${{ matrix.os }}
```

Comment:
Now that no-compat and compat builds are separated in workflows, their artifacts should be named differently to avoid overwriting them